Taskfile.yml

```diff
@@ -143,10 +143,15 @@ tasks:
         PATH="{{.G_NODEJS_14_BIN_DIR}}":$PATH npm install
       - >-
         rsync -a
-        "{{.G_LOG_VIEWER_WEBUI_BUILD_DIR}}/"
-        "{{.OUTPUT_DIR}}/var/www/log_viewer/"
+        "{{.G_LOG_VIEWER_WEBUI_BUILD_DIR}}/client"
+        "{{.OUTPUT_DIR}}/var/www/log_viewer_webui/"
       - |-
-        cd "{{.OUTPUT_DIR}}/var/www/log_viewer/server"
+        cd components/log-viewer-webui/server/
+        rsync -a \
```

Comment:
Just wonder, will the code still work if we first run npm clean-install in another directory and copy what's built into var/www/log_viewer_webui?

Reply:
I think so. However sometimes within node_modules there're symbolic links created during the node dependencies installation, so it's less ideal to copy an initialized node_modules.